db/models/articles.model.js

```diff
@@ -1,11 +1,28 @@
 const db = require('../connection.js');
 
-function selectArticleById(articleId){
-
-    return  fs.readFile(`${__dirname}/../data/owners/${articleId}.json`, "utf-8").then(
-          (fileContents) => {
-            return fileContents;
-          }
-        );
-  }
+exports.selectArticleById = (articleId) => {
+
+    if(isNaN(articleId)){
```

Comment:
What does psql do when you try to pass it an invalid article id? Could we take advantage of this instead of trying to manually check this ourselves? As a rule of thumb, always try making the request and if psql gives you an error to work with, use it - and if it doesn't but you expect to send an error then that's when you need a custom one!